hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml

```diff
@@ -154,10 +154,21 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey.jersey-test-framework</groupId>
-      <artifactId>jersey-test-framework-grizzly2</artifactId>
+      <groupId>org.glassfish.jersey.test-framework</groupId>
+      <artifactId>jersey-test-framework-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+      <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
+      <scope>test</scope>
+      <exclusions>
```

Comment:
shouldn't the exclusions be declared in hadoop-project and picked up everywhere?

Reply:
You’re right. I will make the necessary changes to ensure the POM is concise.